oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -209,28 +209,40 @@ considers the timing of the next `Open edX named release`_.
     periods will vary by the type and scope of the technical change.
 
```

Comment:
TODO: Update/replace the image in this DEPR.

Reply:
1. Github can't display the graphviz. Are we ok with that?
2. There is an image `Removal-Workflow.png`, that also needs to be updated or removed.

Reply:
We use Graphviz elsewhere and I think it's reasonable that most people read the rendered docs on docs.openedx.org. I also think it's pretty easy to read in text.